templates/.travis.yml.mustache

```diff
@@ -1,5 +1,36 @@
 language: nix
 
+docker_install: &DOCKER_INSTALL
```

Comment:
Could you keep `language: nix` and `script:` close to each other since they both describe how to build a Nix package?

Wouldn't it work to use a single `&DOCKER` template that would contain both `install:` and `script:` and possibly also `language: minimal`, `sudo: required`, `services: docker`? Since this provide a solution for building several versions with opam, I think we should also start using a list like `tested_coq_versions` for opam (i.e. have `tested_coq_versions_opam` and `tested_coq_versions_nix`). We could also move the common settings for Nix under their own template and always inherit from a template in the matrix (either the opam or the Nix template).

Reply:
All of these issues should be fixed now in the latest version. It may be a good idea to have an explicit shell script with parameters that invokes `mustache` on the right files?